doc/03-Configuration.md

```diff
@@ -128,6 +128,18 @@ the cluster nodes resynchronise their states after each executed query, and may
 However, this does not necessarily have to be the case if, for instance, Icinga DB is only allowed to connect to a
 single cluster node at a time. This is the case when a load balancer does not randomly route connections to all the
 nodes evenly, but always to the same node until it fails, or if your database cluster nodes have a virtual IP address
-fail over assigned. In such situations, you can set the `wsrep_sync_wait` system variable to `0` in the 
+fail over assigned. In such situations, you can set the `wsrep_sync_wait` system variable to `0` in the
 `/etc/icingadb/config.yml` file to disable it entirely, as Icinga DB doesn't have to wait for cluster
 synchronisation then.
+
+### Systemd Journald Fields
+
+When examining the journal with `journalctl`, fields containing additional information are hidden by default.
+Setting an appropriate
+[`--output` option](https://www.freedesktop.org/software/systemd/man/latest/journalctl.html#Output%20Options)
+will include them, such as: `--output verbose` or `--output json`.
+All Icinga DB fields are prefixed with `ICINGADB_`, e.g., `ICINGADB_ERROR` for error messages.
+
+```
+journalctl --unit icingadb.service --output verbose
+```
```

Comment:
I agree with https://github.com/Icinga/icingadb/pull/743#discussion_r1703623929 in that I'd be happy with just `--output verbose` (i.e. no full command line example).

However, if you add an example, please try to somewhat incorporate it into the text, currently it looks a bit lost at the end. Maybe something like this:

````suggestion
When examining the journal with `journalctl`, fields containing additional information are hidden by default.
Setting an appropriate
[`--output` option](https://www.freedesktop.org/software/systemd/man/latest/journalctl.html#Output%20Options)
such as `--output verbose` or `--output json` will include them, for example:

```
journalctl --unit icingadb.service --output verbose
```

All Icinga DB fields are prefixed with `ICINGADB_`, e.g., `ICINGADB_ERROR` for error messages.
````